lms/templates/index.html

```diff
@@ -53,13 +53,18 @@
 
       % if settings.FEATURES.get('COURSES_ARE_BROWSABLE'):
         <section class="courses">
-            <ul class="courses-listing">
-            %for course in courses:
-              <li class="courses-listing-item">
+            <div class="courses-listing">
```

Comment:
why the move to divs instead of `ul` and 'li`? It really is an list of things...